### PR TITLE
Spring Cloud Azure move builderfactory impl pakcage

### PR DESCRIPTION
**Pull request by saragluna**
To resolve https://github.com/Azure/azure-sdk-for-java/issues/26404.

### CODE REVIEW

File: sdk/spring/spring-cloud-azure-service/src/main/java/com/azure/spring/service/eventhubs/properties/EventBatchProperties.java [hunk 1]
```diff
@@ -3,6 +3,8 @@
 
 package com.azure.spring.service.eventhubs.properties;
 
+import com.azure.spring.service.implementation.eventhubs.properties.EventProcessorClientProperties;
+
```
Comment (backwind1233) [inline sdk/spring/spring-cloud-azure-service/src/main/java/com/azure/spring/service/eventhubs/properties/EventBatchProperties.java:7]: empty line

Comment (saragluna) [reply sdk/spring/spring-cloud-azure-service/src/main/java/com/azure/spring/service/eventhubs/properties/EventBatchProperties.java:7]: There supposed to be empty line between the com.xxx and java.xxx

**Comment by saragluna** [comment]
pipeline passed https://dev.azure.com/azure-sdk/public/_build/results?buildId=1304609&view=results

**Pull request merged**